docs/components/motor/roboclaw.md

```diff
@@ -85,7 +85,7 @@ The following attributes are available for `roboclaw` motors:
 
 | Name | Type | Inclusion | Description |
 | ---- | ---- | --------- | ----------- |
-| `serial_path` | string | **Required** | Serial path of the `roboclaw` controller's USB connection to your robot's filesystem. Run `ls /dev/serial/by-path` in your terminal to find USB serial ports on a Raspberry Pi. <br> Example: `"/dev/serial/by-path/usb-0:1.1:1.0"` |
+| `serial_path` | string | **Required** | The full filesystem path to the serial device, starting with <file>/dev/</file>. With your serial device connected, you can run `sudo dmesg \| grep tty` to show relevant device connection log messages, and then match the returned device name, such as `ttyAMA0` , to its device file, such as <file>/dev/ttyAMA0</file>. On a Raspberry Pi, you can also run `ls /dev/serial/by-path` to list USB serial devices. If you omit this attribute, Viam will attempt to automatically detect the path.<br>Example: `"/dev/serial/by-path/usb-0:1.1:1.0"` |
```

Comment:
Using `AMA0` (usb device) as example, and retaining previous Raspberry Pi-specific guidance. Not replicating elsewhere as we do not want to include Rpi-specific guidance in non-Rpi-specific docs (like component reference pages).